Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM php:8.1.12-cli
+
+RUN apt-get update && apt-get install -y -q git rake ruby-ronn zlib1g-dev && apt-get clean
+
+RUN cd /usr/local/bin && curl -sS https://getcomposer.org/installer | php
+RUN cd /usr/local/bin && mv composer.phar composer
+RUN pecl install grpc
+RUN docker-php-ext-enable grpc
```

Comment:
Everything above this line seems great and is working very well for me.

I'd like to see if we can keep this Dockerfile generic, though, so I have some thoughts on the remaining lines.

I think that in the not-too-distant future (not as part of this ticket or PR), I'd like to publish this image to a public ECR repo, something like `momento-php` or `momento-php-dev` or something.  Then we could re-use it across multiple projects, and devs could pull it without having to do the whole build (which is pretty expensive for the grpc part).

So with that in mind I think we might want to see if we can get rid of the remaining lines after this.  I will add some additional comments below.